include/seqan3/range/views/minimiser.hpp

```diff
@@ -436,7 +428,9 @@ class minimiser_view<urng1_t, urng2_t>::basic_iterator
             advance_window();
         }
         window_values.push_back(window_value());
-        minimiser_value = *std::ranges::min_element(window_values);
+        auto deque_it = std::ranges::min_element(window_values, std::less_equal<value_type>{});
+        minimiser_value = *deque_it;
+        minimiser_position_offset = std::distance(std::begin(window_values), deque_it);
```

Comment:
```suggestion
        auto minimiser_it = std::ranges::min_element(window_values, std::less_equal<value_type>{});
        minimiser_value = *minimiser_it ;
        minimiser_position_offset = std::distance(std::begin(window_values), minimiser_it );
```

Maybe?
`deque_it` doesn't tell me anything :)